docs/specs/preloadDesktopAgents.md

```diff
@@ -0,0 +1,41 @@
+# Preload DA Specification
+
+> Note - The [getAgent() specification](./getAgent.md) relies on Preload DAs behaving as specified in this document. FDC3 apps are now encouraged to use `getAgent()` instead of relying on the existence of the global object.
+
+## Injecting the global FDC3 object
+
+A Preload Desktop Agent MUST provide the FDC3 API via a global accessible as `window.fdc3`. Implementors MAY additionally make the API available through modules, imports, or other means.
+
+The global `window.fdc3` must only be available after the API is ready to use. To enable applications to avoid using the API before it is ready, implementors MUST provide a global `fdc3Ready` event that is fired when the API is ready for use. Implementations should first check for the existence of the FDC3 API and add a listener for this event if it is not found:
```

Comment:
In order that apps work both in electron and on the web, we need to be specifying here that apps use `getAgent()` (or in my view, `fdc3Ready()` too)